web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityListItem.tsx

```diff
@@ -59,11 +58,19 @@ export function IdentityListItem(props: IdentityListItemProps) {
       }}
     >
       <Flex justifyContent="space-between" alignItems="center" width="100%">
-        <Text typography="body2" title={userWithClusterName}>
-          {userWithClusterName}
-        </Text>
+        <Flex flexDirection="column">
+          <Text typography="body2" title={userWithClusterName}>
+            {userWithClusterName}
+          </Text>
```

Comment:
This should be `P2` from the new design system. However, it doesn't exist in the previous branches so backports will have to go back to this format of `<Text typography="body2`. I'll leave it up to you if you wanted to change here and fix in backports or just leave it the way it is.

Reply:
I think I will keep as is to make the backport easier.